src/java.base/share/classes/java/util/EnumMap.java

```diff
@@ -685,9 +685,9 @@ private boolean equals(EnumMap<?,?> em) {
         // Key types match, compare each value
         for (int i = 0; i < keyUniverse.length; i++) {
             Object ourValue =    vals[i];
-            Object hisValue = em.vals[i];
-            if (hisValue != ourValue &&
-                (hisValue == null || !hisValue.equals(ourValue)))
+            Object otherValue = em.vals[i];
+            if (otherValue != ourValue &&
+                (otherValue == null || !otherValue.equals(ourValue)))
```

Comment:
Is this the same as java.util.Objects:  
  `!Objects.equals(vals[i], em.vals[i]);`

Reply:
You are right: we can fold this if-else construct into that utility method. That said, it in *this* PR, I'd prefer not to.

The reason is that I've been preparing a much bigger PR that uses Objects.equals as well as other utility methods and modern language features to improve some equals, hashCode, and compareTo implementations across java.base. I'm planning to publish that PR soon. In fact, this change to Java variable names was cherry-picked from that bigger PR, to separate trivial renames from code changes.